libsplinter/src/biome/rest_api/mod.rs

```diff
@@ -70,10 +70,9 @@ use crate::rest_api::secrets::{AutoSecretManager, SecretManager};
 pub use config::{BiomeRestConfig, BiomeRestConfigBuilder};
 pub use error::BiomeRestResourceManagerBuilderError;
 
-#[cfg(all(feature = "biome-credentials", feature = "rest-api-actix"))]
```

Comment:
Shouldn't be deleted, just add the biome-refresh-tokens guard.

Reply:
rearranged commits so this makes sense